build.sh

```diff
@@ -103,6 +103,7 @@ CMAKE_OPTS+=" -DWORKFLOW_TESTS=${WORKFLOW_BUILD}"
 
 # JCSDA changed test data things, need to make a dummy CRTM directory
 if [[ $BUILD_TARGET == 'hera' ]]; then
+  if [ -d "$dir_root/test-data-release/" ]; then rm -rf $dir_root/test-data-release/; fi
```

Comment:
This is the offending section with the crtm link, I don't understand why the ln points to itself